lib/pengine/pe_output.c

```diff
@@ -1621,6 +1621,106 @@ pe__resource_history_xml(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
+PCMK__OUTPUT_ARGS("resource-list", "pe_working_set_t *", "unsigned int", "gboolean",
+                  "gboolean", "gboolean", "gboolean", "GListPtr", "GListPtr", "gboolean")
+int
+pe__resource_list(pcmk__output_t *out, va_list args)
+{
+    pe_working_set_t *data_set = va_arg(args, pe_working_set_t *);
+    unsigned int print_opts = va_arg(args, unsigned int);
+    gboolean group_by_node = va_arg(args, gboolean);
+    gboolean inactive_resources = va_arg(args, gboolean);
+    gboolean brief_output = va_arg(args, gboolean);
+    gboolean print_summary = va_arg(args, gboolean);
+    GListPtr only_node = va_arg(args, GListPtr);
+    GListPtr only_rsc = va_arg(args, GListPtr);
+    gboolean print_spacer = va_arg(args, gboolean);
+
+    GListPtr rsc_iter;
+    int rc = pcmk_rc_no_output;
+
+    /* If we already showed active resources by node, and
+     * we're not showing inactive resources, we have nothing to do
+     */
+    if (group_by_node && !inactive_resources) {
+        return rc;
+    }
+
+    PCMK__OUTPUT_SPACER_IF(out, print_spacer);
+
+    if (group_by_node) {
+        /* Active resources have already been printed by node */
+        out->begin_list(out, NULL, NULL, "%s", inactive_resources ?  "Inactive Resources" : "");
```

Comment:
Coverity is complaining that inactive_resources can't be 0 here

Reply:
I don't see how this is different from how the code was previously arranged, so I expect coverity only gets run on changed code somehow?  Anyway, I'll just remove the unreachable part.